sandbox/child/index.html

```diff
@@ -15,12 +16,15 @@ <h4>Datadog UI Apps SDK Sandbox: Child</h4>
             client.on('dashboard_cog_menu_context', context => {
                 console.log('Received dashboard cog menu context', context);
 
-                const a = document.createElement('a');
-                a.href = context.shareToken;
-                a.target = '_blank';
-                a.innerHTML = 'Link';
```

Comment:
this no longer works with `sandbox` attr.

Reply:
See my comment below